sync/downstream.py

```diff
@@ -811,6 +812,47 @@ def update_metadata(self, log_files, stability=False):
 
         return disabled
 
+    @mut()
+    def get_gh_metadata(self):
```

Comment:
I think this should be `download_upstream_logs` or something that doesn't contain the string `metadata` since that's really the result of updating from the logs, not the logs themselves.